File: Modul 2/Bahasa c/PRAK205-2310817120001-DinaIzzatiElfadheya.c
#include<stdio.h>
#include<math.h>
void main(){
    int A, B, C, D, E, F, G; 
    printf("Input\n");
    scanf("%i", &A);
    printf("");
    scanf("%i", &B);

    C= pow(B,2)-pow(A,2);
    D= sqrt(C);
    E= (int)A+B+D;
    F= (int)A*D/2;

    printf("\nOutput\n");
    printf("Alas = %i cm \n", D );
    printf("Tinggi = %i cm\n", A );
    printf("Keliling = %i cm\n", E );
    printf("Luas = %i cm^2\n", F);
}

File: Modul 2/Bahasa Python/PRAK205-2310817120001-DinaIzzatiElfadheya.py
import math
print("Input")
input_str= input()
angka= input_str.split()
tinggi, sisi_miring = map(float,angka)
c= (sisi_miring**2)-tinggi**2
alas= math.sqrt(c)
keliling= tinggi+sisi_miring+alas
luas= tinggi*alas/2

print("\nOutput")
print("Alas = ",format(alas,'.0f'), "cm")
print("Tinggi = ",format(tinggi,'.0f'), "cm")
print("Keliling = ",format(keliling,'.0f'),"cm")
print("Luas = ",format(luas,'.0f'), "cm^2")
